avm/res/dev-ops-infrastructure/pool/main.bicep

```
@@ -501,3 +524,40 @@ type managedIdentitiesType = {
   @description('Optional. The resource ID(s) to assign to the resource.')
   userAssignedResourceIds: string[]?
 }?
+
+type standbyAgentsConfigType = {
+  @description('Required. The time at which the agents are needed.')
+  startTime: string
+
+  @description('Required. The time at which the agents are no longer needed.')
+  endTime: string
+
+  @description('Required. The number of agents needed at the start time.')
+  startAgentCount: int
+
+  @description('Required. The number of agents needed at the end time.')
+  endAgentCount: int
+}?
+
+type daysDataType = {
```

```suggestion
@export()
type daysDataType = {
```

Requires a regeneration of the `main.json` file